src/domain/entities/product-tags.ts

```diff
@@ -73,6 +73,88 @@ export class ProductStoresTag extends BaseProductTag {}
 @ProductTag('ingredients_original_tags')
 export class ProductIngredientsOriginalTag extends BaseProductTag {}
 
+// From Issue #22
+@ProductTag('checkers_tags')
+export class ProductCheckersTag extends BaseProductTag {}
+@ProductTag('cities_tags')
+export class ProductCitiesTag extends BaseProductTag {}
+@ProductTag('correctors_tags')
+export class ProductCorrectorsTag extends BaseProductTag {}
+@ProductTag('debug_tags')
+export class ProductDebugTag extends BaseProductTag {}
+@ProductTag('informers_tags')
+export class ProductInformersTag extends BaseProductTag {}
+@ProductTag('ingredients_from_palm_oil_tags')
+export class ProductIngredientsFromPalmOilTag extends BaseProductTag {}
+@ProductTag('ingredients_that_may_be_from_palm_oil_tags')
+export class ProductIngredientsThatMayBeFromPalmOilTag extends BaseProductTag {}
+// Not found in PO
+//@ProductTag('known_nutrients')
+//export class Product Tag extends BaseProductTag {}
+@ProductTag('last_image_dates_tags')
+export class ProductLatestImageDatesTag extends BaseProductTag {}
+// This is a scaler
+//@ProductTag('numbers_of_ingredients')
+//export class Product Tag extends BaseProductTag {}
+@ProductTag('nutrient_levels_tags')
+export class ProductNutrientLevelsTag extends BaseProductTag {}
+// This is a hash
+//@ProductTag('packager_codes')
+//export class Product Tag extends BaseProductTag {}
+@ProductTag('periods_after_opening_tags')
+export class ProductPeriodsAfterOpeningTag extends BaseProductTag {}
+@ProductTag('photographers_tags')
+export class ProductPhotographersTag extends BaseProductTag {}
+@ProductTag('pnns_groups_1_tags')
+export class ProductPnnsGroups1Tag extends BaseProductTag {}
+@ProductTag('pnns_groups_2_tags')
+export class ProductPnnsGroups2Tag extends BaseProductTag {}
+@ProductTag('purchase_places_tags')
+export class ProductPurchasePlacesTag extends BaseProductTag {}
+@ProductTag('unknown_nutrients_tags')
+export class ProductUnknownNutrientsTag extends BaseProductTag {}
+// Thing this is editors
+//@ProductTag('contributors')
```

Comment:
The field in MongoDB is "creator", but it is a scalar, I didn't create an array for it as there's only one creator. (although I did it for things like Nutri-Score...)

Reply:
This is already being loaded as a scalar property on Product